src/main/java/MysqlUserDAO.java

```diff
@@ -20,22 +22,50 @@ public void getUsers() {
 
 	public User getUser(String userName) {
 
-		MysqlDAO mysql = new MysqlUserDAO();
-
 		String sql_query = "SELECT * FROM users WHERE userName=\"" + userName + "\"";
```

Comment:
Not that this matters for this project, but I wonder if [bobby tables](https://xkcd.com/327/) would be an issue here

Reply:
Haha, I think this 'sanitization' of user inputs should be occurring as a preprocessing step before reaching the DAO.